README.md

```diff
@@ -104,7 +104,13 @@ bundle exec rake test:extensions
 bundle exec rake test:all
 ```
 
-It is recommended to run the last command at least once right before pushing.
+### Run absolutely all the tests plus rubocop and bundle audit
+
+```bash
+bundle exec rake
+```
+
+It is recommended to run the last command at least once right before pushing. Also, running this command assures you that what you get locally is the same that you will get in Travis.
```

Comment:
PS, I realise that this is in the pre-existing version but suggesting that someone runs this _at least_ once, right before pushing, is also a bit strange. Why, right before pushing, would they run it twice or more?